docs/plugins/storage.md

```diff
@@ -39,17 +39,33 @@ The plugin has the following optional values:
 
 ### Custom Storage Engine
 You can implement your own storage engine by providing a engine that
```

Comment:
I don't see any reason to list the methods.
Maybe rephrase this to something like that:
```
You can add your own storage engine by implementing the `StorageEngine` interface.
```